docs/cep/reference/extensions/execution/fx.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Function Worker(Fx)
+---
+
+The `fx` extension executes an Edge Worker.
```

Comment:
As a user, I need to understand more about what a function worker is or does. Can you give me a few sentences in this intro explaining what this new thing is and what's in it for me? Try to explain it as you would to a brand new junior developer.

Reply:
Also, at this point we do not know what an Edge Worker is. That documentation has not been written.